startup.py

```diff
@@ -106,7 +106,8 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
         # Flag -P (plugins list file)
         plugins_list_file = self._get_plugins_list_file(exec_path, tk_alias_codename)
         if plugins_list_file:
-            args += " -P {0}".format(plugins_list_file)
+            args += ' -P "{0}'.format(plugins_list_file)
```

Comment:
Are you missing the closing double quote, or should it not be there in this part of the formatted string:
`' -P "{0}'`

Reply:
Hey @staceyoue it is on the next line as cat'ing it after that format call was not working out.

Reply:
Ah, got it!